security/pkg/nodeagent/sds/server.go

```diff
@@ -38,11 +39,12 @@ type Server struct {
 	grpcWorkloadListener net.Listener
 
 	grpcWorkloadServer *grpc.Server
+	stopped            atomic.Bool
```

Comment:
I think this is supposed to be a pointer created by atomic.NewBool?

Reply:
oh yeah